styles/typography/typography-preset-4.json

```diff
@@ -0,0 +1,38 @@
+{
+	"version": 3,
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"title": "Roboto Slab & Manrope",
+	"slug": "typography-preset-4",
+	"styles": {
+		"typography": {
+			"fontFamily": "var:preset|font-family|manrope",
+			"fontSize": "var:preset|font-size|medium",
+			"fontStyle": "normal",
+			"fontWeight": "500",
+			"letterSpacing": "-0.22px",
+			"lineHeight": "1.4"
+		},
+		"blocks": {
+			"core/site-title": {
+				"typography": {
+					"fontFamily": "var:preset|font-family|robot-slab",
```

Comment:
It looks like this one should be manrope. In the designs is manrope and uppercased in `28px`. The strange thing is that I see it in manrope locally.